js/utils.js

```diff
@@ -394,6 +394,14 @@ Fliplet.Registry.set('dynamicListUtils', (function () {
 
     return _.filter(records, function (record) {
       return _.every(filters, function (filter) {
+        if (filter.condition === 'empty') {
+          return _.isEmpty(record.data[filter.column]);
```

Comment:
Numbers will be incorrectly reported as empty using `_.isEmpty()`.

Also, `NaN` is reported as a number using `_.isNumber()`.

This means:

```suggestion
          return _.isEmpty(record.data[filter.column]) && (_.isNaN(record.data[filter.column]) || !_.isNumber(record.data[filter.column]));
```

Also, use variables laid out in https://github.com/Fliplet/fliplet-widget-dynamic-lists/blob/feat/filter-empty/js/utils.js#L397-L400 which will help simplify the code.

Reply:
Fixed. I used isFinite lodash method

Reply:
Didn't know about `_.isFinite()`. Thanks for teaching me something!